### ISSUE TITLE
Does not handle \transpose?

### ISSUE THREAD

**Issue by halvorlu** [aspiers/ly2video#60]
Thanks for a very useful tool to create videos from Lilypond. However, I'm having trouble getting ly2video to handle the \transpose command correctly. Any notes inside the transposed environment are not found in the MIDI file, and therefore ignored. The error is something like

```
grob's pitch 62 not found in midiPitches; probably a tie/ChordName
midiPitches: 52 (E)
Skipping grob and tick
```

I've created the following minimal working example (minimal failing example, that is):

```
\version "2.12.3"

\score {
  \new Staff = "music" {
    \relative c' {
      c1 c1
      \transpose c d {
    d1 d d d
      }
    }
  }
  \layout {}
  \midi {}
}
```

**Comment by aspiers** [comment]
Thanks a lot for the report and minimal test case!  Unfortunately I don't have time to look at this right now, but next time I return to `ly2video` hacking, I'll try to get around to it.  In the mean time, I'd gladly accept pull requests fixing it (and also try to answer questions regarding the codecase).

**Comment by thawk** [comment]
The master should fix this now.

**Comment by aspiers** [comment]
Am I right in thinking that this would have been fixed by #85?

**Comment by aspiers** [comment]
Anyway I just tested and it works now.
